pkg/proxy/iptables/proxier.go

```diff
@@ -229,9 +233,10 @@ func NewProxier(ipt utiliptables.Interface,
 	healthzServer healthcheck.ProxierHealthUpdater,
 	nodePortAddresses []string,
 ) (*Proxier, error) {
-	if utilproxy.ContainsIPv4Loopback(nodePortAddresses) {
+	if localhostNodePorts && utilproxy.ContainsIPv4Loopback(nodePortAddresses) {
 		// Set the route_localnet sysctl we need for exposing NodePorts on loopback addresses
-		klog.InfoS("Setting route_localnet=1, use nodePortAddresses to filter loopback addresses for NodePorts to skip it https://issues.k8s.io/90259")
+		// Refer to https://issues.k8s.io/90259
+		klog.InfoS("Setting route_localnet=1, It allows nodePort services can be accessed via localhost.")
```

Comment:
nit, since we are changing this:

"Setting route_localnet=1; disable iptables.localhostNodePorts (--iptables-localhost-nodeports) or use nodePortAddresses (-- nodeport-addresses) to filter loopback addresses to change this"

Reply:
Thanks,Done!